presto-main/src/main/java/com/facebook/presto/cost/VariableStatsEstimate.java

```diff
@@ -79,6 +83,16 @@ public VariableStatsEstimate(
         checkArgument(distinctValuesCount >= 0 || isNaN(distinctValuesCount), "Distinct values count should be non-negative, got: %s", distinctValuesCount);
         // TODO normalize distinctValuesCount for an empty range (or validate it is already normalized)
         this.distinctValuesCount = distinctValuesCount;
+        this.histogram = histogram;
```

Comment:
Can histogram be null?
Use requireNotNull() in the assignment if null is not expected